Kudu.Contracts/Infrastructure/LockExtensions.cs

```diff
@@ -13,11 +13,12 @@ public static class LockExtensions
         // return true if lock acquired and operation executed
         public static bool TryLockOperation(this IOperationLock lockObj,
```

Comment:
Seems like we do not need this method anymore? since we update most of the code to use LockOperationAsync